src/App.js

```diff
@@ -1,25 +1,29 @@
-import logo from './logo.svg';
-import './App.css';
+import
+{
+  BrowserRouter as Router, Route, Switch, Redirect, Link,
+} from 'react-router-dom';
+import Books from './components/books/Books';
+import Categories from './components/categories/Categories';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <h1>BookStore CMS</h1>
+      <Router>
+        <Link to="/books">Books</Link>
+        <Link to="/categories">Categories</Link>
+        <Switch>
+          <Route path="/books">
+            <Books />
```

Comment:
* I think you can create a component that Renders a form with inputs for the title and the author of the book with a submit button but the button isn't expected to work for now just needs to be rendered
* Refer to project sneak peak 
![image](https://user-images.githubusercontent.com/52497246/130642369-5148d970-6ade-42fa-bec7-49c68f8482d7.png)